.azure/pipelines/ci.yml

```diff
@@ -591,6 +591,7 @@ stages:
 
   - template: jobs/default-build.yml
     parameters:
+      condition: ne(variables['Build.Reason'], 'PullRequest')
```

Comment:
/cc @HaoK is this correct? (I agree the name of this job is odd but isn't it supposed to run in PRs?)

Reply:
Being correct or not is maybe a different question, but I can definitively say that its removal was an accident on my part [in this PR](https://github.com/dotnet/aspnetcore/pull/17663/files#diff-097cdf55e1a931b74fbfe48d7e41c8beL594) (I had removed it for testing in the PR, but forgot to revert). If @HaoK wants to keep it removed anyway that's fine by me.

Reply:
Yeah I noticed this was accidentally removed recently too, its good to add this back